RBS.Auth.WebApi/RBS.Auth.WebApi/appsettings.json

```diff
@@ -1,16 +1,19 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Information",
-            "Microsoft": "Warning",
-            "Microsoft.Hosting.Lifetime": "Information"
-        }
-    },
-    "Auth":{
-        "Issuer": "authServer",
-        "Audience": "resourceServer",
-        "Secret": "secret123123keysurpa",
-        "TokenLifetime": "7200"
-    },
+  "ConnectionStrings": {
+    "AuthDb": "Server=(LocalDB)\\MSSQLLocalDB;Database=RBS.Auth;Trusted_Connection=True;MultipleActiveResultSets=true"
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "Auth": {
+    "Issuer": "authServer",
+    "Audience": "resourceServer",
+    "Secret": "secret123123keysurpa",
```

Comment:
Really, do you want to store JWT signature secret in app settings?

Reply:
This change will be in the next iteration